src/main/app/src/app/informatie-objecten/informatie-object-create-attended/informatie-object-create-attended.component.ts

```diff
@@ -0,0 +1,263 @@
+          template.formControl.setValue(null); // Always reset selected template after group change or clearing
+          if (selectedTemplateGroup) {
+            this.sjabloonOptions$.next(selectedTemplateGroup.templates);
+          } else {
+            this.sjabloonOptions$.next([]);
+          }
+        },
+      ),
+    );
+
+    this.subscriptions$.push(
+      template.formControl.valueChanges.subscribe((selectedTemplate) => {
+        if (selectedTemplate && selectedTemplate.informatieObjectTypeUUID) {
+          const infoObjectType = this.informatieObjectTypes.find(
+            (type) => type.uuid === selectedTemplate.informatieObjectTypeUUID,
+          );
+
+          if (infoObjectType) {
+            informationObjectType.formControl.setValue(
+              infoObjectType.omschrijving || null,
+            );
+            confidentiality.formControl.setValue(
+              this.translateService.instant(
+                this.vertrouwelijkaanduidingToTranslationKeyPipe.transform(
+                  infoObjectType.vertrouwelijkheidaanduiding,
+                ) || null,
+              ),
+            );
+
+            return;
+          }
+        }
+        informationObjectType.formControl.setValue(null);
+        confidentiality.formControl.setValue(null);
+      }),
+    );
+  }
+
+  private async fetchInformatieobjecttypes(): Promise<any> {
+    try {
+      const informatieobjecttypes = await firstValueFrom(
+        this.informatieObjectenService
+          .listInformatieobjecttypes(this.zaak.zaaktype.uuid)
+          .pipe(first()),
+      );
+      return informatieobjecttypes;
+    } catch (error) {
+      console.error("Error fetching informatieobjecttypes:", error);
+    }
+  }
+
+  ngOnDestroy(): void {
+    for (const subscription of this.subscriptions$) {
+      subscription.unsubscribe();
+    }
+  }
+
+  onFormSubmit(formGroup: FormGroup): void {
+    if (formGroup) {
+      const documentCreateData = new DocumentCreationData();
+      Object.keys(formGroup.controls).forEach((key) => {
+        const control = formGroup.controls[key];
+        const value = control.value;
+
+        // Convert form fields to REST end point Body Parameters
+        switch (key) {
+          case "templateGroup":
+            documentCreateData["zaakUuid"] = this.zaak.uuid;
+            documentCreateData["smartDocumentsTemplateGroupId"] = value.id;
+            break;
+          case "template":
+            documentCreateData["smartDocumentsTemplateId"] = value.id;
+            break;
+          case "informationObjectType":
+          case "confidentiality":
+            // Fields not end point Body Parameters; 'just informational', so leave them out. End point will determine these values itself (again)
+            break;
+          default:
+            if (value instanceof moment) {
+              documentCreateData[key] = value; // conversie niet nodig, ISO-8601 in UTC gaat goed met java ZonedDateTime.parse
```

Comment:
Is this comment really needed? Partly in Dutch now..

Reply:
Well, it helps me to understand the document create accepts a moment as well; so I can let go ;-)

Reply:
Removed anyway; since I now know ;-)